.github/workflows/test.yml

```diff
@@ -19,9 +19,12 @@ jobs:
         with:
           go-version-file: go.mod
 
-      - name: Run tests
+      - name: Check headers and go mod
```

Comment:
The `ci` target includes `test` again. Should we move all dependent targets except for `test` into a new target that we call from here?

Alternatively, we could retain just the old structure. I don't have strong opinions on either.

Reply:
I removed the `test` from the `ci` target 🙂 https://github.com/digitalocean/digitalocean-cloud-controller-manager/pull/712/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R38

but yes i agree we could just retain the old structure

Reply:
Sorry missed that, thanks!